### DISCUSSION TITLE
Report warnings?

### DISCUSSION THREAD

**Discussion by boxmastersystems** [mpdf/mpdf#1982] [General]
Hi folks, I'm new to github. I'm developing in "strict mode", and exiting on all warnings, deprecations etc.  Is it appropriate to raise warnings as bugs, or submit changes to eradicate them?

**Comment by finwe** [comment]
Yes, feel free to report warnings, please. Just count on that not all of them can be fixed right away as development time is scarce and thorough fixes are preferred.
